libminifi/include/properties/Properties.h

```diff
@@ -82,6 +84,16 @@ class Properties {
    */
   int getInt(const std::string &key, int default_value) const;
 
+  utils::optional<std::string> get(const std::string& key) const {
+    std::string result;
+    const bool found = get(key, result);
```

Comment:
Just fancy, for me this is the case when ternary operator simplifies the code:
```
return get(key, result) ? result : utils::nullopt;
```

Feel free to ignore in case you doesn't share this opinion.

Reply:
I totally agree but they need to be of the same type to be able to use the ternary operator and in this case I rely on conversions from different types to construct the return value.

Reply:
```
/home/szaszm/nifi-minifi-cpp-2/libminifi/include/properties/Properties.h: In member function ‘nonstd::optional_lite::optional<std::__cxx11::basic_string<char> > org::apache::nifi::minifi::Properties::get(const string&) const’:
/home/szaszm/nifi-minifi-cpp-2/libminifi/include/properties/Properties.h:89:29: error: operands to ‘?:’ have different types ‘std::string’ {aka ‘std::__cxx11::basic_string<char>’} and ‘const nonstd::optional_lite::nullopt_t’
   89 |     return get(key, result) ? result : utils::nullopt;
      |            ~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
😞